.github/workflows/5_builderpackage_reporting.yml

```diff
@@ -59,13 +64,13 @@ jobs:
         run: echo "version=$(jq -r .version<VERSION.json)" >> "$GITHUB_OUTPUT"
 
       - name: Build with Gradle
-        run: ./gradlew build -Dversion=${{ steps.version.outputs.version }} -Drevision=${{ inputs.revision }}
+        run: ./gradlew build -Dversion=${{ steps.version.outputs.version }} -Drevision=${{ inputs.revision || '0' }}
```

Comment:
The default value for `inputs.revision` is 0, so it will never be null or undefined.

```yaml
    inputs:
      revision:
        description: "Revision"
        type: string
        default: "0"
```